### PR TITLE
update config docs and config file for prod

### PR DESCRIPTION
**Pull request by sanfern**
Signed-off-by: sferna1 <santhosh.fernandes@gmail.com>

### CODE REVIEW

**Review by charleskbliu0 — APPROVED** [review]
Looks good.

File: docs/configdoc.md [hunk 1]
```diff
@@ -8,11 +8,9 @@ See [l3afd.cfg](https://github.com/l3af-project/l3afd/blob/main/config/l3afd.cfg
...
 pid-file: ./l3afd.pid
-datacenter: dummy
+datacenter: dc
 bpf-dir: /dev/shm
 bpf-log-dir:
-kernel-major-version: 4
-kernel-minor-version: 15
 shutdown-timeout: 1s
 http-client-timeout: 10s
 max-nf-restart-count: 3
```
Comment (jniesz) [inline docs/configdoc.md:16]: This needs to get updated as well

**Review by jniesz — APPROVED** [review]

File: kf/bpf.go [hunk 2]
```diff
@@ -581,6 +581,9 @@ func (b *BPF) GetArtifacts(conf *config.Config) error {
 		}
 	case fileScheme:
 		{
+			if isDefaultURLUsed {
+				URL.Path = path.Join(URL.Path, b.Program.Name, b.Program.Version, platform, b.Program.Artifact)
```
Comment (jniesz) [inline kf/bpf.go:585]: Doesn't it make sense to move this logic above the switch statement after line 551?

**Review by jniesz — APPROVED** [review]
LGTM

**Review by charleskbliu0 — APPROVED** [review]
Looks good

**Review by jaysheth2 — APPROVED** [review]
LGTM

**Pull request merged**